### PR TITLE
Feature/306 fix questions submission endpoints 1

### PR DESCRIPTION
**Pull request by mmnoo**
# Description

- point  monitoring forms to right endpoint and modify payload
- make dated required since they are used to identify monitoring forms in the UI list

Fixes # 306

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: mrtt-ui/src/language.js
Comment (mmnoo) [inline mrtt-ui/src/language.js:23]: added `required`, just reordered the rest

File: mrtt-ui/src/components/SiteInterventions/SiteInterventions.js
Comment (mmnoo) [inline mrtt-ui/src/components/SiteInterventions/SiteInterventions.js:144]: this is going to screw up the `successCallback` and will be addressed in upcoming prs

Comment (mmnoo) [reply mrtt-ui/src/components/SiteInterventions/SiteInterventions.js:144]: (I wont merge until all the prs for this are approved and its bug free in totality, just need to break up for my own sanity)

File: mrtt-ui/src/components/Costs/CostsForm.js
Comment (mmnoo) [inline mrtt-ui/src/components/Costs/CostsForm.js:134]: this is going to screw up the `successCallback` and will be addressed in upcoming prs

Comment (mmnoo) [reply mrtt-ui/src/components/Costs/CostsForm.js:134]: REthinking using different urls for this............ we should just stick to registration answers for all of them. Angelo suggested that was an option. I didng think it was a big deal until I got my hands dirty wit this.

Comment (mmnoo) [reply mrtt-ui/src/components/Costs/CostsForm.js:134]: https://sparkgeo.slack.com/archives/C036D4TLVPT/p1663377679361169

File: mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js
Comment (mmnoo) [inline mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js:42]: made dates required because they are used as an identifier for the form in the UI list

**Comment by julia-rom** [comment]
file conflicts here from recent merge

**Review by julia-rom — APPROVED** [review]

**Pull request merged**
